bin/fin

```diff
@@ -6292,6 +6293,11 @@ case "$1" in
 			_exec wp "$@"
 		fi
 		;;
+	composer)
+		load_configuration
```

Comment:
Sorry, missed it. There is no need to load configuration. Actually it is counterproductive o do it, because it will throw in case it does not find a project.

Reply:
@achekulaev I that is the main purpose to run similar to drush and wp-cli where it runs inside of the projects container therefore it needs to make sure the project exists before hand.

Reply:
Right. My bad.